source/slang/hlsl.meta.slang

```diff
@@ -20806,3 +20806,294 @@ T workgroupUniformLoad<T>(__ref T v)
         return v;
     }
 }
+
+//
+// Pack/Unpack Math Intrinsics
+//
+
+typealias uint8_t4_packed = uint32_t;
+typealias int8_t4_packed = uint32_t;
+
+[__readNone]
```

Comment:
Use `//@hidden:` here to hide the following functions from stdlib reference: https://shader-slang.com/stdlib-reference/global-decls/index.html